CONTRIBUTING.md

```diff
@@ -2,51 +2,57 @@
 
-- Raise Issues on either the functionality or documentation
-- Submit Changes to Improve Documentation 
-- Submit Proposals for new Features/Enhancements 
-- Submit Changes to the Source Code
+However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover what we're looking for. By addressing all the points we're looking for, it raises the chances we can quickly merge or address your contributions.
+
+That said, OpenEBS is an innovation in open-source. You are welcome to contribute in any way you can and all the help provided is very much appreciated. 
+
+- Raise issues to request new functionality, fix documentation or for reporting bugs.
+- Submit changes to improve documentation. 
+- Submit proposals for new features/enhancements.
+- Submit fixes for bugs reported/found. 
+- Submit changes to the source code.
+- Solve existing Issues.
 
 There are just a few simple guidelines that you need to follow before providing your hacks. 
 
 ## Raising Issues
 
-When Raising issues, please specify the following:
-- Setup details (like hyperconverged/dedicated), orchestration engine - kubernetes, docker swarm etc,. 
-- Scenario where the issue was seen to occur
+When raising issues, please specify the following:
+- Setup details (like hyperconverged/dedicated), orchestration engine - kubernetes, docker swarm etc. 
+- Scenario where the issue was seen to occur (With details on how to reproduce it)
 - If the issue is with storage, include maya version, maya osh-status and maya omm-status.
 - Errors and log messages that are thrown by the software
 
-
 ## Submit Change to Improve Documentation
 
-Getting Documentation Right is Hard! Please raise a PR with you proposed changes. 
+Getting documentation right is hard! Refer to this [page](./contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md) page for more information on you could improve the developer documentation by submitting pull requests with appropriate tags. Here's a [list of tags](./contribute/labels-of-issues.md) that could be used. 
+
 
 ## Submit Proposals for New Features
 
-There is always something more that is required, to make it easier to suit your use-cases. Feel free to join the discussion on the features or raise a new PR with your proposed change. 
+There is always something more that is required, to make it easier to suit your use-cases. Feel free to join the discussion on new features or raise a PR with your proposed change. 
 
-## Contributing to Source Code
+## Contributing to Source Code & Bug Fixes
 
-Provide PRs for bug fixes or enhancements.
+Provide PRs with appropriate tags for bug fixes or enhancements to the source code. For a list of tags that could be used, take a look [here]((./contribute/labels-of-issues.md)).
```

Comment:
take a look = see

Reply:
I am converting it to 
For list of tags that could be used, see this.